File: Aula21_26_11_2020/Py/ex3.py
ip = int(input("Para descobrir a classe do IPv4 desejado, digite o primeiro octeto: "))

if ip >= 1 and ip <= 127:
    print('Sua classe de IPv4 é A')
elif ip >= 128 and ip <=191:
    print("Sua classe de IPv4 é B")
else:
    print("Sua classe de IPv4 é C")

File: Aula21_26_11_2020/Cs/ex3.cs
// using System;

// namespace Cs
// {
//     class ex3
//     {
//         static void Main(string[] args)
//         {
//             int ip;

//             Console.WriteLine("Digite o primeiro octeto do IPv4 para saber sua classe: "); 
//             ip = Convert.ToInt32(Console.ReadLine());

//             if (ip >= 1 && ip <= 127)
//             {
//                 Console.WriteLine("Sua classe de IPv4 é A");
//             }
//             else if (ip >= 128 && ip <= 191)
//             {
//                 Console.WriteLine("Sua classe de IPv4 é B");
//             }
//             else
//             {
//                 Console.WriteLine("Sua classe de IPv4 é C");
//             }
//         }
//     }
// }